server/src/test/java/org/elasticsearch/action/search/TransportMultiSearchActionTests.java

```diff
@@ -39,12 +39,9 @@
 import org.elasticsearch.tasks.Task;
```

Comment:
This change doesn't seem to be for this pr ?

Reply:
Yes, I pulled this out to https://github.com/elastic/elasticsearch/pull/48523 and didn't delete it here.